dockers/docker-router-advertiser/wait_for_intf.sh.j2

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+function wait_until_iface_ready
+{
+    IFACE=$1
+
+    echo "Waiting until interface $IFACE is up..."
+
+    # Wait for the interface to come up (i.e., 'ip link show' returns 0)
+    until ip link show dev $IFACE up > /dev/null 2>&1; do
+        sleep 1
+    done
+
+    echo "Interface $IFACE is up"
+
+    echo "Waiting until interface $IFACE has an IPv4 address..."
+
+    # Wait until the interface gets assigned an IPv4 address
+    while true; do
+        IP=$(ip -4 addr show dev $IFACE | grep "inet " | awk '{ print $2 }' | cut -d '/' -f1)
+
+        if [ -n "$IP" ]; then
+            break
+        fi
+
+        sleep 1
+    done
+
+    echo "Interface $IFACE is configured with IP $IP"
+}
+
+
+# Wait for all interfaces to come up and have IPv4 addresses assigned
+{% for (name, prefix) in VLAN_INTERFACE %}
+wait_until_iface_ready {{ name }}
```

Comment:
Just checking why only vlan_interfaces?

Reply:
We currently configure the T0s to send router advertisements out to devices under our VLANs. More interfaces can be added in the future, along with more flexible configuration. Do you see any issue with this?